aten/src/ATen/core/jit_type.h

```diff
@@ -1199,7 +1199,7 @@ template <typename T>
 struct getTypePtr_ final {
   static TypePtr call() {
     if (!isCustomClassRegistered<T>()) {
-      throw c10::Error("Type could not be converted to any of the known types.", "");
+      throw c10::Error(std::string() + "Type " + typeid(T).name() + " could not be converted to any of the known types.", "");
```

Comment:
Oy! RTTI!!

Reply:
good catch. Seems I forgot to remove my debug code...

I'm actually hoping to enable this after the gcc5 switch using compile time type identification,  having a better error message here would be really nice, but that's separate from this PR.